features/fixtures/TestApp/config/config.yml

```diff
@@ -36,7 +39,26 @@ dunglas_api:
                 default:               3
                 client_can_change:     true
 
```

Comment:
This configuration option is [false by default, as in the Configuration class](https://github.com/dunglas/DunglasApiBundle/blob/master/DependencyInjection/Configuration.php#L43).
There is nothing such as a compiler pass or extension changing it to true, but only [the extension loading](https://github.com/dunglas/DunglasApiBundle/blob/master/DependencyInjection/DunglasApiExtension.php#L73-L76) the [fos_user.xml](https://github.com/dunglas/DunglasApiBundle/blob/master/Resources/config/fos_user.xml) config file in the container.

However, fos user support was automatically enabled before, but has been removed in https://github.com/dunglas/DunglasApiBundle/commit/586b1caa5cd4cb02fa7f55fa0e55a7dbe116ef51#diff-6b1fc2ed2ec34e06eb4189fc43c60824L33